src/library-view-qt/library-view-qt.h

```diff
@@ -0,0 +1,57 @@
+#pragma once
+
+#include <QBoxLayout>
+#include <QLineEdit>
+#include <QPointer>
+#include <QWidget>
+
+#include <libaudcore/i18n.h>
+#include <libaudcore/playlist.h>
+#include <libaudcore/plugin.h>
+#include <libaudcore/preferences.h>
+
+#define CFG_ID "library-view"
+
+constexpr auto DEFAULT_LIBRARY_PATH = "~";
+constexpr auto DEFAULT_PLAYLIST_TITLE = "Library view";
+constexpr auto DEFAULT_PLAYING_TITLE = "Playing";
```

Comment:
If you use a temporary playlist, Audacious will already call it "Now playing".  Please do not add additional playlists for temporary use.